.github/workflows/ci.yml

```diff
@@ -191,7 +191,6 @@ jobs:
         with:
           args: .
         env:
-          GITHUB_PUSH_BEFORE_SHA: ${{ github.event.before }}
```

Comment:
I can find several references of this variable, e.g. [here](https://github.com/GitGuardian/ggshield/blob/main/ggshield/core/git_hooks/ci/commit_range.py#L196). Are you sure this is not used ?

Reply:
Yes for sca and iac, ggshield scan does not depend on GITHUB_PUSH_BEFORE_SHA  (following https://github.com/GitGuardian/ggshield/pull/902 )

Reply:
Ah yes I did not remember this change ! Good for me then !